### DISCUSSION TITLE
Feature request: Add a bottom preview pane.

### DISCUSSION THREAD

**Discussion by ChipZhang** [FossifyOrg/General-Discussion#412] [Ideas]
Add an option to display a bottom preview pane.

So that, when viewing a photo in non full screen mode, it displays a pane in the bottom, with the thumbnails of all pictures in the current folder. And the user can click the thumbnail to quickly switch to the desired photo.

Sorry I can't upload an image right now to better describe this feature.
If anyone wants to know more about what this feature is, Aves Gallery app has this feature.
You can install it (available in Play and Fdroid), then go to Settings -> Viewer -> Overlay -> check 'Show thumbnails', then close the settings, and open a photo in an album, and a bottom preview pane appears.

**Comment by Aga-C** [comment]
Please create a [feature request](https://github.com/FossifyOrg/Gallery/issues/new?assignees=&labels=feature+request%2Cneeds+triage&projects=&template=feature_request.yml) for this.
